sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -358,25 +371,33 @@ def wrapper(*args, **kwargs):
 
   wrapper.__name__ = name
   wrapper.__doc__ = (
-      f":meth:`pandas.{base_type.__name__}.{name}` is not supported in the "
-      f"Beam DataFrame API {reason_data['explanation']}")
+      f":meth:`{_prettify_pandas_type(base_type)}.{name}` is not supported in "
```

Comment:
Should we say not yet supported, or otherwise give an indication that this is subject to change?